### DISCUSSION TITLE
add referrerpolicy ?

### DISCUSSION THREAD

**Discussion by sharadagarwal** [alshedivat/al-folio#916] [Q&A] [answered]
I'm using ACM's Author-Izer Service, which allows visitors to my website to access my papers free of charge. However, for it to work properly, I need to add referrerpolicy to my PDF links. For example:

`<a href="https://dl.acm.org/doi/10.1145/3544216.3544234?cid=81100632765" referrerpolicy="no-referrer-when-downgrade">
`

Is there a way to do this in the bib file?

**Comment by cmoralesmx** [comment, marked as answer]
Currently it is not possible to add this attribute `referrerpolicy="no-referrer-when-downgrade"` to the generated link.
But, you can make a couple of changes to make this possible. 

This change would require renaming a field in the `bib` file and creating a new entry in the template which creates the links.

1. In your `.bib` file: Assuming you are using the `html` field to store the link to your papers, you'd rename the `html` field as `ownhtml`, as shown here,
![image](https://user-images.githubusercontent.com/1790200/194739216-3a2958cb-7303-4f7b-b237-09db7dd68b5d.png)

1. In `_layouts/bib.html`: You'd duplicate the 3 lines corresponding to the entry for `entry.html` (around line 141). 
And in the 3 lines that you just duplicated, change `if entry.html` -> `if entry.ownhtml` and add your custom attribute, as shown here,
![image](https://user-images.githubusercontent.com/1790200/194739575-054bded0-0687-4b4c-95fe-0be0ddfbb173.png)

After making these changes, you'd need to rename the `html` field to `ownhtml` in each of the bib entries you want to share in the same way.
Doing so allows you to keep using the basic `html` field for other publications while having a new custom entry to suit your need.

**Reply by sharadagarwal** [reply]
Thank you @cmoralesmx ! This worked !
